src/definition/fetch_test.go

```diff
@@ -99,14 +98,14 @@ func TestTransformBypassesError(t *testing.T) {
 	}
 
 	v, err := Transform(FromRaw("metric_name_3"), transformFunc)("nonExistingGroup", "entity2", raw)
-	assert.EqualError(t, err, "group not found")
+	assert.EqualError(t, err, "group \"nonExistingGroup\" not found")
 	assert.Nil(t, v)
 
 	v, err = Transform(FromRaw("metric_name_3"), transformFunc)("group1", "nonExistingEntity", raw)
-	assert.EqualError(t, err, "entity not found")
+	assert.EqualError(t, err, "entity \"nonExistingEntity\" not found")
 	assert.Nil(t, v)
 
 	v, err = Transform(FromRaw("non_existing_metric"), transformFunc)("group1", "entity2", raw)
-	assert.EqualError(t, err, "metric not found")
+	assert.EqualError(t, err, "metric \"non_existing_metric\" not found")
 	assert.Nil(t, v)
 }
```

Comment:
As above, seems easy to build a table for this

Reply:
I think creating a struct with 5 fields here brings a little benefit here, right now test cases are very clear and readable. Little abstraction is worse than little duplication. We could wrap each case in sub-test, but it seems out of scope for this already big PR. I only touched areas of the code I was debugging. Perhaps there shouldn't be tests on `FromRaw` level at all if we take a look more into it.